Duo256m/Alpine/README.md

```diff
@@ -32,26 +36,39 @@ last_update: 2024-09-29
 
 ## Installation Steps
 
-### Download Alpine minirootfs and Debian image
+### Method A: Flash community image
+
+Download the prebuilt community image: https://drive.google.com/file/d/1zhhB6AdgvjjuzBWjY6TchdX5b0uNWzP-/view
```

Comment:
Where is this image published? Community forum/GitHub links please?

Reply:
idk, it's provided by @Arielfoever ;)